doc/ws.md

```diff
@@ -456,7 +456,7 @@ Removes an event listener emulating the `EventTarget` interface.
 
 ### websocket.send(data[, options][, callback])
 
-- `data` {Any} The data to send.
+- `data` {number | array | arrayBuffer | buffer | object | string} The data to send.
```

Comment:
```suggestion
- `data` {Array|Number|Object|String|ArrayBuffer|Buffer|DataView|TypedArray} The data to send.
```

Can you please change it also for `websocket.ping()` and `websocket.pong()`?

Reply:
From what I understand, `DataView` is pretty much just an `ArrayBuffer`, though with a specific task, and `TypedArray` isn't actually an accessible property/constructor. Does it really make sense to add them?

Will add to `ping()` and `pong()`.

Reply:
> From what I understand, `DataView` is pretty much just an `ArrayBuffer`, though with a specific task, and `TypedArray` isn't actually an accessible property/constructor. Does it really make sense to add them?

Yes it does

```
> var buffer = new ArrayBuffer(2);
undefined
> var view = new DataView(buffer);
undefined
> Buffer.from(view)
<Buffer >
> Buffer.from(view.buffer)
<Buffer 00 00>
```